lib/explorer/backend/lazy_frame.ex

```diff
@@ -21,7 +22,7 @@ defmodule Explorer.Backend.LazyFrame do
   @doc false
   def new(df) do
     Explorer.Backend.DataFrame.new(
-      %__MODULE__{names: df.names, dtypes: df.dtypes},
+      %__MODULE__{names: df.names, dtypes: df.dtypes, original_data: df.data},
```

Comment:
maybe we should store only the backend?

Reply:
done in 3a8d825716b5b0ee1646388d99ca0565939557e8